core/src/main/scala/org/broadinstitute/dsde/rawls/entities/EntityManager.scala

```diff
@@ -74,12 +74,12 @@ class EntityManager(providerBuilders: Set[EntityProviderBuilder[_ <: EntityProvi
 object EntityManager {
   def defaultEntityManager(dataSource: SlickDataSource, workspaceManagerDAO: WorkspaceManagerDAO,
                            dataRepoDAO: DataRepoDAO, samDAO: SamDAO, bqServiceFactory: GoogleBigQueryServiceFactory,
-                           config: DataRepoEntityProviderConfig, cacheEnabled: Boolean)
+                           config: DataRepoEntityProviderConfig, cacheEnabled: Boolean, metricsPrefix: String = "")
```

Comment:
is this good practice or should i make another method? see also: [this](core/src/main/scala/org/broadinstitute/dsde/rawls/entities/local/LocalEntityProvider.scala)